falco/templates/daemonset.yaml

```diff
@@ -60,6 +60,10 @@ spec:
             - --cri
             - /run/containerd/containerd.sock
             {{- end }}
+            {{- if .Values.crio.enabled }}
+            - --cri
+            - /run/crio/crio.sock
+            {{- end }}
```

Comment:
What's the expected behavior when both `crio` and `containerd` are enabled?

Reply:
I'd wish for falco to handle clusters with mixed runtimes. I can test if this works as expected. I ran it with both set to true with no errors but only with crio nodes.

Reply:
AFAIK, Falco does not handle multiple `--cri` options (I believe just the last one will take effect).
Thus, I believe we have to handle that here in some way, too. Not sure how, atm.

Reply:
What of the existing chart where docker and containerd are both set to true by default? Should it all be consolidated into one map rather than a set of true/false values?

Reply:
AFAIK, since Falco does require a cli flag for Docker, both Docker and containerd options can work simultaneously.
So, it's a different problem. 

Unfortunately, it's already easy for users to get confused by those options.

Reply:
Ok. After I confirm these factors I'll throw in some input validation stating only one can be enabled over the other for now.